### ISSUE TITLE
Add Disabled property to aws_route53_health_check

### ISSUE THREAD

**Issue by Glen-Moonpig** [terraform-providers/terraform-provider-aws#8937]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

[CreateHealthCheck](https://docs.aws.amazon.com/Route53/latest/APIReference/API_CreateHealthCheck.html) supports creating a disabled health check, but this is not implemented in aws_route53_health_check. I want to be able to disable health checks in certain environments I am deploying to, to avoid unnecessary costs.

### New or Affected Resource(s)

aws_route53_health_check

### Potential Terraform Configuration

```hcl
resource "aws_route53_health_check" "example" {
  disabled = "true"
}
```

### References

* #0000

**Comment by nvmsteve** [comment]
This issue appears to have been 'duplicated' multiple times, this suggests a number of people would like this feature. This one has been open for almost a year.
There are 2 pull requests (at least) to add this functionality including #10436 which was closed because it was submitted after #8939, which was closed due to inactivity.

Does anyone know how  we can we progress getting this functionality added?

**Comment by Glen-Moonpig** [comment]
> This issue appears to have been 'duplicated' multiple times, this suggests a number of people would like this feature. This one has been open for almost a year.
> There are 2 pull requests (at least) to add this functionality including #10436 which was closed because it was submitted after #8939, which was closed due to inactivity.
> 
> Does anyone know how we can we progress getting this functionality added?

It would be very useful for us to have this functionality added, but adding such basic functionality to the terraform provider is a real pain and takes considerable time, effort and skill. It would be great if AWS could just generate the code based on their API spec..

**Comment by anGie44** [comment]
This feature been merged and will release with `v3.2.0` of the Terraform AWS Provider, expected out later today. Thanks @Glen-Moonpig for your contributions!

**Comment by Glen-Moonpig** [comment]
> This feature been merged and will release with `v3.2.0` of the Terraform AWS Provider, expected out later today. Thanks @Glen-Moonpig for your contributions!

Thanks, glad to see this attribute finally ready to go 😅
